src/frontend/src/pages/CreditsPage/CreditsPage.tsx

```diff
@@ -26,7 +26,8 @@ const CreditsPage: React.FC = () => {
     { name: 'Megan Liu', color: '#3d609e' },
     { name: 'Peyton McKee', color: '#6a3941' },
     { name: 'Ryan Howe', color: '#3cce20' },
-    { name: 'Nezam Jazayeri', color: '#8a8f5c' }
+    { name: 'Nezam Jazayeri', color: '#8a8f5c' },
+    { name: 'Ethan Szeto', color: '#404040'}
   ];
```

Comment:
Your prettier is failing because you forgot to add a space before your closing bracket! If you set up your VSCode to run prettier on save, it should fix this for you when you save your file!